app/validators/aus_phone_number_validator.rb

```diff
@@ -0,0 +1,7 @@
+class AusPhoneNumberValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    if value != nil && value !~ /\A04[0-9]{8}\z/
```

Comment:
Agreed, although I think there's a whole flow around setting up phone numbers and authenticator details that's likely going to be left for another PR. I think we will address this validator concern as part of that.